open-reac/src/main/java/com/powsybl/openreac/parameters/output/network/VscNetworkOutput.java

```diff
@@ -17,17 +17,13 @@
  */
 public class VscNetworkOutput extends AbstractNetworkOutput<VscConverterStationModification> {
     private static final String ELEMENT = "vsc_converter_stations";
+    public static final int EXPECTED_COLS = 8;
     private static final int ID_COLUMN_INDEX = 1;
-
```

Comment:
You have removed this FIXME? So no issue or you have the answer?

Reply:
For me no issue. In the export of AMPL code, there is no column V. Only target V. And the values used for export are V[n] for bus n (the only value available).